webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/policies/TestNodeAndGlobalParameterLookup.scala

```diff
@@ -249,6 +271,59 @@ class TestNodeAndGlobalParameterLookup extends Specification {
       test(all(_), """${something . cfengine}""", List(CharSeq("${something . cfengine"), CharSeq("}")))
     }
 
+    "parse a valid engine" in {
+      test(
+        all(_)
+        , """${rudder.engine[foo]("tst")}"""
```

Comment:
I'm not sure we want quote around "tst". But it can wait for the moment where we harden syntaxe & extend to be able to pass json

Reply:
Do we want : `${engine.X[premiere-value][sous-champ-value] | option1 = xxx }` ?